src/forms/Predictions.js

```diff
@@ -14,26 +14,25 @@ import { BenefitsLineGraph } from './output/BenefitsLineGraph';
 // ========================================
 // COMPONENTS
 // ========================================
-/** @todo description
-*
-* @function
-* @param {object} props Values described below
-* @property {object} props.future Client future/predictive data.
-* @property {string} props.time Used in class names. Meant to make
-*     this more easily decoupled in future.
-* @property {function} props.updateClientValue Update client state
-*     values.
-*
-* @returns {class} Component
-*/
+/** @todo Cash flow row for trying out different future incomes.
+ *
+ * @function
+ * @param {object} props
+ * @property {object} props.future Client future/predictive data.
+ * @property {string} props.time Used in class names. Meant to make
+ *     this more easily decoupled in future.
+ * @property {function} props.updateClientValue Update client state
+ *     value.
+ * @property {object} props.snippets Language-specific text
+ * @note As per Project Hope's input, for the first prototype
```

Comment:
These should all be @param, not @property. That way they show up as sub-objects of `props`.